### ISSUE TITLE
compilation errors

### ISSUE THREAD

**Issue by shrinidhi-venkatakrishnan** [Netflix/vmaf#835]
FAILED: test/test_feature_extractor.p/.._src_opt.c.o 
ccache cc -Itest/test_feature_extractor.p -Itest -I../test -Iinclude -I../include -Isrc -I../src -fdiagnostics-color=always -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -std=c11 -O3 -D_GNU_SOURCE -MD -MQ test/test_feature_extractor.p/.._src_opt.c.o -MF test/test_feature_extractor.p/.._src_opt.c.o.d -o test/test_feature_extractor.p/.._src_opt.c.o -c ../src/opt.c
cc: error: unrecognized command line option ‘-fdiagnostics-color=always’
[2/139] ccache cc -Itest/test_feature_extractor.p -Itest -I../test -Iinclude -I../include -Isrc -I../src -fdiagnostics-color=always -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -std=c11 -O3 -D_GNU_SOURCE -MD -MQ test/test_feature_extractor.p/.._src_dict.c.o -MF test/test_feature_extractor.p/.._src_dict.c.o.d -o test/test_feature_extractor.p/.._src_dict.c.o -c ../src/dict.c
FAILED: test/test_feature_extractor.p/.._src_dict.c.o 
ccache cc -Itest/test_feature_extractor.p -Itest -I../test -Iinclude -I../include -Isrc -I../src -fdiagnostics-color=always -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -std=c11 -O3 -D_GNU_SOURCE -MD -MQ test/test_feature_extractor.p/.._src_dict.c.o -MF test/test_feature_extractor.p/.._src_dict.c.o.d -o test/test_feature_extractor.p/.._src_dict.c.o -c ../src/dict.c
cc: error: unrecognized command line option ‘-fdiagnostics-color=always’
[3/139] /prj/video_sys/apps4/People/shrinidh/tests/ll/bin/meson --internal vcstagger ../include/vcs_version.h.in include/vcs_version.h 2.1.1 /prj/video_sys/apps4/People/shrinidh/tests/vmaf-master/libvmaf/include @VCS_TAG@ '(.*)' /usr/bin/git --git-dir /prj/video_sys/apps4/People/shrinidh/tests/vmaf-master/libvmaf/../.git describe --tags --long --match '?.*.*' --always
fatal: not a git repository: '/prj/video_sys/apps4/People/shrinidh/tests/vmaf-master/libvmaf/../.git'
ninja: build stopped: subcommand failed.
Makefile:2: recipe for target 'all' failed
make: *** [all] Error 1

How do i fix this issue?

**Comment by slhck** [comment]
How did you download the repository? I suggest you clone it as a Git repo; it seems that a test is failing since it can't find the `.git` directory.

**Comment by kylophone** [comment]
> How did you download the repository? I suggest you clone it as a Git repo; it seems that a test is failing since it can't find the `.git` directory.

Yes, that's correct.

**Comment by kfeng19** [comment]
Also having a compilation error here:
```
ninja: Entering directory `libvmaf/build'
[1/93] cc -Itest/test_model.p -Itest -I../test -Iinclude -I../include -Isrc -I../src -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -std=c11 -O3 -D_GNU_SOURCE -pthread -pedantic -DOC_NEW_STYLE_INCLUDES -MD -MQ test/test_model.p/meson-generated_.._.._src_vmaf_b_v0.6.3.json.c.o -MF test/test_model.p/meson-generated_.._.._src_vmaf_b_v0.6.3.json.c.o.d -o test/test_model.p/meson-generated_.._.._src_vmaf_b_v0.6.3.json.c.o -c src/vmaf_b_v0.6.3.json.c
FAILED: test/test_model.p/meson-generated_.._.._src_vmaf_b_v0.6.3.json.c.o 
cc -Itest/test_model.p -Itest -I../test -Iinclude -I../include -Isrc -I../src -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -std=c11 -O3 -D_GNU_SOURCE -pthread -pedantic -DOC_NEW_STYLE_INCLUDES -MD -MQ test/test_model.p/meson-generated_.._.._src_vmaf_b_v0.6.3.json.c.o -MF test/test_model.p/meson-generated_.._.._src_vmaf_b_v0.6.3.json.c.o.d -o test/test_model.p/meson-generated_.._.._src_vmaf_b_v0.6.3.json.c.o -c src/vmaf_b_v0.6.3.json.c
src/vmaf_b_v0.6.3.json.c:1:42: warning: ISO C forbids empty initializer braces [-Wpedantic]
 unsigned char src_vmaf_b_v0_6_3_json[] = {
                                          ^
src/vmaf_b_v0.6.3.json.c:1:15: error: zero or negative size array ‘src_vmaf_b_v0_6_3_json’
 unsigned char src_vmaf_b_v0_6_3_json[] = {
               ^
[2/93] cc -Itest/test_model.p -Itest -I../test -Iinclude -I../include -Isrc -I../src -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -std=c11 -O3 -D_GNU_SOURCE -pthread -pedantic -DOC_NEW_STYLE_INCLUDES -MD -MQ test/test_model.p/meson-generated_.._.._src_vmaf_4k_v0.6.1.json.c.o -MF test/test_model.p/meson-generated_.._.._src_vmaf_4k_v0.6.1.json.c.o.d -o test/test_model.p/meson-generated_.._.._src_vmaf_4k_v0.6.1.json.c.o -c src/vmaf_4k_v0.6.1.json.c
FAILED: test/test_model.p/meson-generated_.._.._src_vmaf_4k_v0.6.1.json.c.o 
cc -Itest/test_model.p -Itest -I../test -Iinclude -I../include -Isrc -I../src -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -std=c11 -O3 -D_GNU_SOURCE -pthread -pedantic -DOC_NEW_STYLE_INCLUDES -MD -MQ test/test_model.p/meson-generated_.._.._src_vmaf_4k_v0.6.1.json.c.o -MF test/test_model.p/meson-generated_.._.._src_vmaf_4k_v0.6.1.json.c.o.d -o test/test_model.p/meson-generated_.._.._src_vmaf_4k_v0.6.1.json.c.o -c src/vmaf_4k_v0.6.1.json.c
src/vmaf_4k_v0.6.1.json.c:1:43: warning: ISO C forbids empty initializer braces [-Wpedantic]
 unsigned char src_vmaf_4k_v0_6_1_json[] = {
                                           ^
src/vmaf_4k_v0.6.1.json.c:1:15: error: zero or negative size array ‘src_vmaf_4k_v0_6_1_json’
 unsigned char src_vmaf_4k_v0_6_1_json[] = {
               ^
[3/93] cc -Itest/test_model.p -Itest -I../test -Iinclude -I../include -Isrc -I../src -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -std=c11 -O3 -D_GNU_SOURCE -pthread -pedantic -DOC_NEW_STYLE_INCLUDES -MD -MQ test/test_model.p/meson-generated_.._.._src_vmaf_float_v0.6.1neg.json.c.o -MF test/test_model.p/meson-generated_.._.._src_vmaf_float_v0.6.1neg.json.c.o.d -o test/test_model.p/meson-generated_.._.._src_vmaf_float_v0.6.1neg.json.c.o -c src/vmaf_float_v0.6.1neg.json.c
FAILED: test/test_model.p/meson-generated_.._.._src_vmaf_float_v0.6.1neg.json.c.o 
cc -Itest/test_model.p -Itest -I../test -Iinclude -I../include -Isrc -I../src -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -std=c11 -O3 -D_GNU_SOURCE -pthread -pedantic -DOC_NEW_STYLE_INCLUDES -MD -MQ test/test_model.p/meson-generated_.._.._src_vmaf_float_v0.6.1neg.json.c.o -MF test/test_model.p/meson-generated_.._.._src_vmaf_float_v0.6.1neg.json.c.o.d -o test/test_model.p/meson-generated_.._.._src_vmaf_float_v0.6.1neg.json.c.o -c src/vmaf_float_v0.6.1neg.json.c
src/vmaf_float_v0.6.1neg.json.c:1:49: warning: ISO C forbids empty initializer braces [-Wpedantic]
 unsigned char src_vmaf_float_v0_6_1neg_json[] = {
                                                 ^
src/vmaf_float_v0.6.1neg.json.c:1:15: error: zero or negative size array ‘src_vmaf_float_v0_6_1neg_json’
 unsigned char src_vmaf_float_v0_6_1neg_json[] = {
               ^
[4/93] cc -Itest/test_model.p -Itest -I../test -Iinclude -I../include -Isrc -I../src -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -std=c11 -O3 -D_GNU_SOURCE -pthread -pedantic -DOC_NEW_STYLE_INCLUDES -MD -MQ test/test_model.p/meson-generated_.._.._src_vmaf_float_v0.6.1.json.c.o -MF test/test_model.p/meson-generated_.._.._src_vmaf_float_v0.6.1.json.c.o.d -o test/test_model.p/meson-generated_.._.._src_vmaf_float_v0.6.1.json.c.o -c src/vmaf_float_v0.6.1.json.c
FAILED: test/test_model.p/meson-generated_.._.._src_vmaf_float_v0.6.1.json.c.o 
cc -Itest/test_model.p -Itest -I../test -Iinclude -I../include -Isrc -I../src -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -std=c11 -O3 -D_GNU_SOURCE -pthread -pedantic -DOC_NEW_STYLE_INCLUDES -MD -MQ test/test_model.p/meson-generated_.._.._src_vmaf_float_v0.6.1.json.c.o -MF test/test_model.p/meson-generated_.._.._src_vmaf_float_v0.6.1.json.c.o.d -o test/test_model.p/meson-generated_.._.._src_vmaf_float_v0.6.1.json.c.o -c src/vmaf_float_v0.6.1.json.c
src/vmaf_float_v0.6.1.json.c:1:46: warning: ISO C forbids empty initializer braces [-Wpedantic]
 unsigned char src_vmaf_float_v0_6_1_json[] = {
                                              ^
src/vmaf_float_v0.6.1.json.c:1:15: error: zero or negative size array ‘src_vmaf_float_v0_6_1_json’
 unsigned char src_vmaf_float_v0_6_1_json[] = {
               ^
[5/93] cc -Itest/test_model.p -Itest -I../test -Iinclude -I../include -Isrc -I../src -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -std=c11 -O3 -D_GNU_SOURCE -pthread -pedantic -DOC_NEW_STYLE_INCLUDES -MD -MQ test/test_model.p/meson-generated_.._.._src_vmaf_v0.6.1neg.json.c.o -MF test/test_model.p/meson-generated_.._.._src_vmaf_v0.6.1neg.json.c.o.d -o test/test_model.p/meson-generated_.._.._src_vmaf_v0.6.1neg.json.c.o -c src/vmaf_v0.6.1neg.json.c
FAILED: test/test_model.p/meson-generated_.._.._src_vmaf_v0.6.1neg.json.c.o 
cc -Itest/test_model.p -Itest -I../test -Iinclude -I../include -Isrc -I../src -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -std=c11 -O3 -D_GNU_SOURCE -pthread -pedantic -DOC_NEW_STYLE_INCLUDES -MD -MQ test/test_model.p/meson-generated_.._.._src_vmaf_v0.6.1neg.json.c.o -MF test/test_model.p/meson-generated_.._.._src_vmaf_v0.6.1neg.json.c.o.d -o test/test_model.p/meson-generated_.._.._src_vmaf_v0.6.1neg.json.c.o -c src/vmaf_v0.6.1neg.json.c
src/vmaf_v0.6.1neg.json.c:1:43: warning: ISO C forbids empty initializer braces [-Wpedantic]
 unsigned char src_vmaf_v0_6_1neg_json[] = {
                                           ^
src/vmaf_v0.6.1neg.json.c:1:15: error: zero or negative size array ‘src_vmaf_v0_6_1neg_json’
 unsigned char src_vmaf_v0_6_1neg_json[] = {
               ^
[6/93] cc -Itest/test_model.p -Itest -I../test -Iinclude -I../include -Isrc -I../src -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -std=c11 -O3 -D_GNU_SOURCE -pthread -pedantic -DOC_NEW_STYLE_INCLUDES -MD -MQ test/test_model.p/meson-generated_.._.._src_vmaf_float_b_v0.6.3.json.c.o -MF test/test_model.p/meson-generated_.._.._src_vmaf_float_b_v0.6.3.json.c.o.d -o test/test_model.p/meson-generated_.._.._src_vmaf_float_b_v0.6.3.json.c.o -c src/vmaf_float_b_v0.6.3.json.c
FAILED: test/test_model.p/meson-generated_.._.._src_vmaf_float_b_v0.6.3.json.c.o 
cc -Itest/test_model.p -Itest -I../test -Iinclude -I../include -Isrc -I../src -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -std=c11 -O3 -D_GNU_SOURCE -pthread -pedantic -DOC_NEW_STYLE_INCLUDES -MD -MQ test/test_model.p/meson-generated_.._.._src_vmaf_float_b_v0.6.3.json.c.o -MF test/test_model.p/meson-generated_.._.._src_vmaf_float_b_v0.6.3.json.c.o.d -o test/test_model.p/meson-generated_.._.._src_vmaf_float_b_v0.6.3.json.c.o -c src/vmaf_float_b_v0.6.3.json.c
src/vmaf_float_b_v0.6.3.json.c:1:48: warning: ISO C forbids empty initializer braces [-Wpedantic]
 unsigned char src_vmaf_float_b_v0_6_3_json[] = {
                                                ^
src/vmaf_float_b_v0.6.3.json.c:1:15: error: zero or negative size array ‘src_vmaf_float_b_v0_6_3_json’
 unsigned char src_vmaf_float_b_v0_6_3_json[] = {
               ^
[7/93] cc -Itest/test_model.p -Itest -I../test -Iinclude -I../include -Isrc -I../src -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -std=c11 -O3 -D_GNU_SOURCE -pthread -pedantic -DOC_NEW_STYLE_INCLUDES -MD -MQ test/test_model.p/meson-generated_.._.._src_vmaf_float_4k_v0.6.1.json.c.o -MF test/test_model.p/meson-generated_.._.._src_vmaf_float_4k_v0.6.1.json.c.o.d -o test/test_model.p/meson-generated_.._.._src_vmaf_float_4k_v0.6.1.json.c.o -c src/vmaf_float_4k_v0.6.1.json.c
FAILED: test/test_model.p/meson-generated_.._.._src_vmaf_float_4k_v0.6.1.json.c.o 
cc -Itest/test_model.p -Itest -I../test -Iinclude -I../include -Isrc -I../src -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -std=c11 -O3 -D_GNU_SOURCE -pthread -pedantic -DOC_NEW_STYLE_INCLUDES -MD -MQ test/test_model.p/meson-generated_.._.._src_vmaf_float_4k_v0.6.1.json.c.o -MF test/test_model.p/meson-generated_.._.._src_vmaf_float_4k_v0.6.1.json.c.o.d -o test/test_model.p/meson-generated_.._.._src_vmaf_float_4k_v0.6.1.json.c.o -c src/vmaf_float_4k_v0.6.1.json.c
src/vmaf_float_4k_v0.6.1.json.c:1:49: warning: ISO C forbids empty initializer braces [-Wpedantic]
 unsigned char src_vmaf_float_4k_v0_6_1_json[] = {
                                                 ^
src/vmaf_float_4k_v0.6.1.json.c:1:15: error: zero or negative size array ‘src_vmaf_float_4k_v0_6_1_json’
 unsigned char src_vmaf_float_4k_v0_6_1_json[] = {
               ^
[8/93] cc -Itest/test_model.p -Itest -I../test -Iinclude -I../include -Isrc -I../src -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -std=c11 -O3 -D_GNU_SOURCE -pthread -pedantic -DOC_NEW_STYLE_INCLUDES -MD -MQ test/test_model.p/meson-generated_.._.._src_vmaf_v0.6.1.json.c.o -MF test/test_model.p/meson-generated_.._.._src_vmaf_v0.6.1.json.c.o.d -o test/test_model.p/meson-generated_.._.._src_vmaf_v0.6.1.json.c.o -c src/vmaf_v0.6.1.json.c
FAILED: test/test_model.p/meson-generated_.._.._src_vmaf_v0.6.1.json.c.o 
cc -Itest/test_model.p -Itest -I../test -Iinclude -I../include -Isrc -I../src -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -std=c11 -O3 -D_GNU_SOURCE -pthread -pedantic -DOC_NEW_STYLE_INCLUDES -MD -MQ test/test_model.p/meson-generated_.._.._src_vmaf_v0.6.1.json.c.o -MF test/test_model.p/meson-generated_.._.._src_vmaf_v0.6.1.json.c.o.d -o test/test_model.p/meson-generated_.._.._src_vmaf_v0.6.1.json.c.o -c src/vmaf_v0.6.1.json.c
src/vmaf_v0.6.1.json.c:1:40: warning: ISO C forbids empty initializer braces [-Wpedantic]
 unsigned char src_vmaf_v0_6_1_json[] = {
                                        ^
src/vmaf_v0.6.1.json.c:1:15: error: zero or negative size array ‘src_vmaf_v0_6_1_json’
 unsigned char src_vmaf_v0_6_1_json[] = {
               ^
[9/93] /home/OVT/kaijun.feng/miniconda3/envs/vmaf/bin/meson --internal vcstagger ../include/vcs_version.h.in include/vcs_version.h 2.1.1 /home/OVT/kaijun.feng/Desktop/vmaf/libvmaf/include @VCS_TAG@ '(.*)' /bin/git --git-dir /home/OVT/kaijun.feng/Desktop/vmaf/libvmaf/../.git describe --tags --long --match '?.*.*' --always
[10/93] c++ -Itest/test_model.p -Itest -I../test -Iinclude -I../include -Isrc -I../src -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wnon-virtual-dtor -Wextra -std=c++11 -O3 -pthread -pedantic -DOC_NEW_STYLE_INCLUDES -MD -MQ test/test_model.p/.._src_svm.cpp.o -MF test/test_model.p/.._src_svm.cpp.o.d -o test/test_model.p/.._src_svm.cpp.o -c ../src/svm.cpp
FAILED: test/test_model.p/.._src_svm.cpp.o 
c++ -Itest/test_model.p -Itest -I../test -Iinclude -I../include -Isrc -I../src -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wnon-virtual-dtor -Wextra -std=c++11 -O3 -pthread -pedantic -DOC_NEW_STYLE_INCLUDES -MD -MQ test/test_model.p/.._src_svm.cpp.o -MF test/test_model.p/.._src_svm.cpp.o.d -o test/test_model.p/.._src_svm.cpp.o -c ../src/svm.cpp
../src/svm.cpp: In instantiation of ‘SVMModelParser<TSource>::SVMModelParser(TSource&&) [with TSource = SVMModelParserFileSource]’:
../src/svm.cpp:3162:65:   required from here
../src/svm.cpp:3023:79: error: use of deleted function ‘SVMModelParserFileSource::SVMModelParserFileSource(SVMModelParserFileSource&&)’
  SVMModelParser(TSource&& model_source) : model_source(std::move(model_source)) {}
                                                                               ^
../src/svm.cpp:2950:7: note: ‘SVMModelParserFileSource::SVMModelParserFileSource(SVMModelParserFileSource&&)’ is implicitly deleted because the default definition would be ill-formed:
 class SVMModelParserFileSource {
       ^
../src/svm.cpp:2950:7: error: use of deleted function ‘std::basic_ifstream<char>::basic_ifstream(const std::basic_ifstream<char>&)’
In file included from ../src/svm.cpp:11:0:
/usr/include/c++/4.8.2/fstream:427:11: note: ‘std::basic_ifstream<char>::basic_ifstream(const std::basic_ifstream<char>&)’ is implicitly deleted because the default definition would be ill-formed:
     class basic_ifstream : public basic_istream<_CharT, _Traits>
           ^
/usr/include/c++/4.8.2/fstream:427:11: error: use of deleted function ‘std::basic_istream<char>::basic_istream(const std::basic_istream<char>&)’
In file included from /usr/include/c++/4.8.2/fstream:38:0,
                 from ../src/svm.cpp:11:
/usr/include/c++/4.8.2/istream:58:11: note: ‘std::basic_istream<char>::basic_istream(const std::basic_istream<char>&)’ is implicitly deleted because the default definition would be ill-formed:
     class basic_istream : virtual public basic_ios<_CharT, _Traits>
           ^
/usr/include/c++/4.8.2/istream:58:11: error: use of deleted function ‘std::basic_ios<char>::basic_ios(const std::basic_ios<char>&)’
In file included from /usr/include/c++/4.8.2/ios:44:0,
                 from /usr/include/c++/4.8.2/istream:38,
                 from /usr/include/c++/4.8.2/fstream:38,
                 from ../src/svm.cpp:11:
/usr/include/c++/4.8.2/bits/basic_ios.h:66:11: note: ‘std::basic_ios<char>::basic_ios(const std::basic_ios<char>&)’ is implicitly deleted because the default definition would be ill-formed:
     class basic_ios : public ios_base
           ^
In file included from /usr/include/c++/4.8.2/ios:42:0,
                 from /usr/include/c++/4.8.2/istream:38,
                 from /usr/include/c++/4.8.2/fstream:38,
                 from ../src/svm.cpp:11:
/usr/include/c++/4.8.2/bits/ios_base.h:786:5: error: ‘std::ios_base::ios_base(const std::ios_base&)’ is private
     ios_base(const ios_base&);
     ^
In file included from /usr/include/c++/4.8.2/ios:44:0,
                 from /usr/include/c++/4.8.2/istream:38,
                 from /usr/include/c++/4.8.2/fstream:38,
                 from ../src/svm.cpp:11:
/usr/include/c++/4.8.2/bits/basic_ios.h:66:11: error: within this context
     class basic_ios : public ios_base
           ^
In file included from ../src/svm.cpp:11:0:
/usr/include/c++/4.8.2/fstream:427:11: error: use of deleted function ‘std::basic_ios<char>::basic_ios(const std::basic_ios<char>&)’
     class basic_ifstream : public basic_istream<_CharT, _Traits>
           ^
/usr/include/c++/4.8.2/fstream:427:11: error: use of deleted function ‘std::basic_filebuf<char>::basic_filebuf(const std::basic_filebuf<char>&)’
/usr/include/c++/4.8.2/fstream:72:11: note: ‘std::basic_filebuf<char>::basic_filebuf(const std::basic_filebuf<char>&)’ is implicitly deleted because the default definition would be ill-formed:
     class basic_filebuf : public basic_streambuf<_CharT, _Traits>
           ^
In file included from /usr/include/c++/4.8.2/ios:43:0,
                 from /usr/include/c++/4.8.2/istream:38,
                 from /usr/include/c++/4.8.2/fstream:38,
                 from ../src/svm.cpp:11:
/usr/include/c++/4.8.2/streambuf:802:7: error: ‘std::basic_streambuf<_CharT, _Traits>::basic_streambuf(const std::basic_streambuf<_CharT, _Traits>&) [with _CharT = char; _Traits = std::char_traits<char>]’ is private
       basic_streambuf(const basic_streambuf& __sb)
       ^
In file included from ../src/svm.cpp:11:0:
/usr/include/c++/4.8.2/fstream:72:11: error: within this context
     class basic_filebuf : public basic_streambuf<_CharT, _Traits>
           ^
../src/svm.cpp: In instantiation of ‘SVMModelParser<TSource>::SVMModelParser(TSource&&) [with TSource = SVMModelParserBufferSource]’:
../src/svm.cpp:3168:74:   required from here
../src/svm.cpp:3023:79: error: use of deleted function ‘SVMModelParserBufferSource::SVMModelParserBufferSource(SVMModelParserBufferSource&&)’
  SVMModelParser(TSource&& model_source) : model_source(std::move(model_source)) {}
                                                                               ^
../src/svm.cpp:2983:7: note: ‘SVMModelParserBufferSource::SVMModelParserBufferSource(SVMModelParserBufferSource&&)’ is implicitly deleted because the default definition would be ill-formed:
 class SVMModelParserBufferSource {
       ^
../src/svm.cpp:2983:7: error: use of deleted function ‘std::basic_istringstream<char>::basic_istringstream(const std::basic_istringstream<char>&)’
In file included from ../src/svm.cpp:12:0:
/usr/include/c++/4.8.2/sstream:272:11: note: ‘std::basic_istringstream<char>::basic_istringstream(const std::basic_istringstream<char>&)’ is implicitly deleted because the default definition would be ill-formed:
     class basic_istringstream : public basic_istream<_CharT, _Traits>
           ^
/usr/include/c++/4.8.2/sstream:272:11: error: use of deleted function ‘std::basic_istream<char>::basic_istream(const std::basic_istream<char>&)’
/usr/include/c++/4.8.2/sstream:272:11: error: use of deleted function ‘std::basic_ios<char>::basic_ios(const std::basic_ios<char>&)’
/usr/include/c++/4.8.2/sstream:272:11: error: use of deleted function ‘std::basic_stringbuf<char>::basic_stringbuf(const std::basic_stringbuf<char>&)’
/usr/include/c++/4.8.2/sstream:64:11: note: ‘std::basic_stringbuf<char>::basic_stringbuf(const std::basic_stringbuf<char>&)’ is implicitly deleted because the default definition would be ill-formed:
     class basic_stringbuf : public basic_streambuf<_CharT, _Traits>
           ^
In file included from /usr/include/c++/4.8.2/ios:43:0,
                 from /usr/include/c++/4.8.2/istream:38,
                 from /usr/include/c++/4.8.2/fstream:38,
                 from ../src/svm.cpp:11:
/usr/include/c++/4.8.2/streambuf:802:7: error: ‘std::basic_streambuf<_CharT, _Traits>::basic_streambuf(const std::basic_streambuf<_CharT, _Traits>&) [with _CharT = char; _Traits = std::char_traits<char>]’ is private
       basic_streambuf(const basic_streambuf& __sb)
       ^
In file included from ../src/svm.cpp:12:0:
/usr/include/c++/4.8.2/sstream:64:11: error: within this context
     class basic_stringbuf : public basic_streambuf<_CharT, _Traits>
           ^
ninja: build stopped: subcommand failed.
make: *** [all] Error 1
```

**Comment by nm1002** [comment]
> Also having a compilation error here (cloned the repo):
> 
> ```
> ninja: Entering directory `libvmaf/build'
> [1/93] cc -Itest/test_model.p -Itest -I../test -Iinclude -I../include -Isrc -I../src -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -std=c11 -O3 -D_GNU_SOURCE -pthread -pedantic -DOC_NEW_STYLE_INCLUDES -MD -MQ test/test_model.p/meson-generated_.._.._src_vmaf_b_v0.6.3.json.c.o -MF test/test_model.p/meson-generated_.._.._src_vmaf_b_v0.6.3.json.c.o.d -o test/test_model.p/meson-generated_.._.._src_vmaf_b_v0.6.3.json.c.o -c src/vmaf_b_v0.6.3.json.c
> FAILED: test/test_model.p/meson-generated_.._.._src_vmaf_b_v0.6.3.json.c.o 
> cc -Itest/test_model.p -Itest -I../test -Iinclude -I../include -Isrc -I../src -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -std=c11 -O3 -D_GNU_SOURCE -pthread -pedantic -DOC_NEW_STYLE_INCLUDES -MD -MQ test/test_model.p/meson-generated_.._.._src_vmaf_b_v0.6.3.json.c.o -MF test/test_model.p/meson-generated_.._.._src_vmaf_b_v0.6.3.json.c.o.d -o test/test_model.p/meson-generated_.._.._src_vmaf_b_v0.6.3.json.c.o -c src/vmaf_b_v0.6.3.json.c
> src/vmaf_b_v0.6.3.json.c:1:42: warning: ISO C forbids empty initializer braces [-Wpedantic]
>  unsigned char src_vmaf_b_v0_6_3_json[] = {
>                                           ^
> src/vmaf_b_v0.6.3.json.c:1:15: error: zero or negative size array ‘src_vmaf_b_v0_6_3_json’
>  unsigned char src_vmaf_b_v0_6_3_json[] = {
>                ^
> [2/93] cc -Itest/test_model.p -Itest -I../test -Iinclude -I../include -Isrc -I../src -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -std=c11 -O3 -D_GNU_SOURCE -pthread -pedantic -DOC_NEW_STYLE_INCLUDES -MD -MQ test/test_model.p/meson-generated_.._.._src_vmaf_float_v0.6.1.json.c.o -MF test/test_model.p/meson-generated_.._.._src_vmaf_float_v0.6.1.json.c.o.d -o test/test_model.p/meson-generated_.._.._src_vmaf_float_v0.6.1.json.c.o -c src/vmaf_float_v0.6.1.json.c
> FAILED: test/test_model.p/meson-generated_.._.._src_vmaf_float_v0.6.1.json.c.o 
> cc -Itest/test_model.p -Itest -I../test -Iinclude -I../include -Isrc -I../src -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -std=c11 -O3 -D_GNU_SOURCE -pthread -pedantic -DOC_NEW_STYLE_INCLUDES -MD -MQ test/test_model.p/meson-generated_.._.._src_vmaf_float_v0.6.1.json.c.o -MF test/test_model.p/meson-generated_.._.._src_vmaf_float_v0.6.1.json.c.o.d -o test/test_model.p/meson-generated_.._.._src_vmaf_float_v0.6.1.json.c.o -c src/vmaf_float_v0.6.1.json.c
> src/vmaf_float_v0.6.1.json.c:1:46: warning: ISO C forbids empty initializer braces [-Wpedantic]
>  unsigned char src_vmaf_float_v0_6_1_json[] = {
>                                               ^
> src/vmaf_float_v0.6.1.json.c:1:15: error: zero or negative size array ‘src_vmaf_float_v0_6_1_json’
>  unsigned char src_vmaf_float_v0_6_1_json[] = {
> ...
> /usr/include/c++/4.8.2/sstream:272:11: error: use of deleted function ‘std::basic_istream<char>::basic_istream(const std::basic_istream<char>&)’
> /usr/include/c++/4.8.2/sstream:272:11: error: use of deleted function ‘std::basic_ios<char>::basic_ios(const std::basic_ios<char>&)’
> /usr/include/c++/4.8.2/sstream:272:11: error: use of deleted function ‘std::basic_stringbuf<char>::basic_stringbuf(const std::basic_stringbuf<char>&)’
> /usr/include/c++/4.8.2/sstream:64:11: note: ‘std::basic_stringbuf<char>::basic_stringbuf(const std::basic_stringbuf<char>&)’ is implicitly deleted because the default definition would be ill-formed:
>      class basic_stringbuf : public basic_streambuf<_CharT, _Traits>
>            ^
> In file included from /usr/include/c++/4.8.2/ios:43:0,
>                  from /usr/include/c++/4.8.2/istream:38,
>                  from /usr/include/c++/4.8.2/fstream:38,
>                  from ../src/svm.cpp:11:
> /usr/include/c++/4.8.2/streambuf:802:7: error: ‘std::basic_streambuf<_CharT, _Traits>::basic_streambuf(const std::basic_streambuf<_CharT, _Traits>&) [with _CharT = char; _Traits = std::char_traits<char>]’ is private
>        basic_streambuf(const basic_streambuf& __sb)
>        ^
> In file included from ../src/svm.cpp:12:0:
> /usr/include/c++/4.8.2/sstream:64:11: error: within this context
>      class basic_stringbuf : public basic_streambuf<_CharT, _Traits>
>            ^
> ninja: build stopped: subcommand failed.
> make: *** [all] Error 1
> ```

I also have the question,did you fix it?
